src/api/java/com/minecolonies/api/colony/requestsystem/requestable/Food.java

```diff
@@ -73,7 +103,21 @@ public static Food deserialize(final IFactoryController controller, final Compou
     {
         final int count = compound.getInt(NBT_COUNT);
         final ItemStack result = compound.keySet().contains(NBT_RESULT) ? ItemStackUtils.deserializeFromNBT(compound.getCompound(NBT_RESULT)) : ItemStackUtils.EMPTY;
+        final List<ItemStorage> items = new ArrayList<>();
+
+        if (compound.contains(NBT_EXCLUSION))
+        {
+            final ListNBT filterableItems = compound.getList(NBT_EXCLUSION, Constants.NBT.TAG_COMPOUND);
+            for (int i = 0; i < filterableItems.size(); ++i)
+            {
+                items.add(new ItemStorage(ItemStack.read(filterableItems.getCompound(i))));
+            }
+        }
 
+        if (!items.isEmpty())
+        {
```

Comment:
this check is not necessary, else we hand it an empty list which isn't a problem either.